cpskin/core/viewlets/footer.pt

```diff
@@ -9,7 +9,7 @@ if (ploneFormTabbing) {
       <span tal:condition="context/@@is_gdpr"><a href=""
                 tal:attributes="href string:${view/navigation_root_url}/gdpr-view"
                 i18n:translate="Mentions L&eacute;gales">Mentions L&eacute;gales</a> - </span>
-        <span i18n:translate="" omit-tag="">Site r&eacute;alis&eacute; en collaboration avec</span>&nbsp;<a href="http://www.imio.be" target="_blank">IMIO</a>&nbsp;<tal:block i18n:translate="sous licence">sous licence</tal:block>
+        <span i18n:translate="" omit-tag="">Site r&eacute;alis&eacute; en collaboration avec</span>&nbsp;<a href="http://www.imio.be" target="_blank" i18n:ignore="">IMIO</a>&nbsp;<tal:block i18n:translate="sous licence">sous licence</tal:block>
```

Comment:
i18n:ignore ?
I don't know this one, and it seems[ travis also don't like is](https://travis-ci.org/github/IMIO/cpskin.core/builds/678221992)

Reply:
I removed the "i18n:ignore" i18nude syntax